### DISCUSSION TITLE
How To detected new message from whatsapp

### DISCUSSION THREAD

**Discussion by jupiterzhuo** [amanharwara/altus#130] [Q&A]
Hello Amanharwara, 

Great for your amazing work. i want to ask how you detected new message from whatsapp? cause i have one small project from my office to warp internal chat using electron. Thanks

**Comment by amanharwara** [comment]
If you are looking to do some sort of automation, then Altus isn't really suitable for that. You might be better off using one of the many WhatsApp APIs available on Github
